src/ui/public/courier/saved_object/saved_object.js

```diff
@@ -55,11 +55,10 @@ export default function SavedObjectFactory(es, kbnIndex, Promise, Private, Notif
     let customInit = config.init || _.noop;
 
     // optional search source which this object configures
-    self.searchSource = config.searchSource && new SearchSource();
+    self.searchSource = config.searchSource ? new SearchSource() : null;
```

Comment:
I'd like to avoid a `null` vs `undefined` debate, and I can see we're using `null` in a million places in the codebase, so I don't really care about it's use here. Unfortunately we're also using loose equality to compare values against it, which renders it a bit moot. 😄 

But I would like to know if `null` has a role in any logic in this context? E.g. `if (someVal === nulll) doSomething()`. I agree with you if a "nothing" value actually matters... but if not, then ¯\_(ツ)_/¯ Seems odd but I don't really care.